src/sas/qtgui/Perspectives/Fitting/FittingUtilities.py

```diff
@@ -542,7 +542,7 @@ def residualsData1D(reference_data, current_data, weights):
             pass
 
     residuals.x = current_data.x[index][0]
-    residuals.dy = numpy.ones(len(residuals.y))
+    residuals.dy = None
```

Comment:
I have no idea but there are a lot of instances of Data1D.x, .dx, .etc being set to None so just doing a "monkey see/monkey do."  If it would be better coding practice to not state ... and it really does default to None that would be fine by me.

Reply:
An explicit setting of something doesn't hurt, I was just wondering if there was a specific reason to do so in this case. Leaving it set to None will ensure the values are always what they are meant to be.